src/test/java/uk/nhs/ctp/testhelper/matchers/FhirMatchers.java

```diff
@@ -41,6 +42,22 @@
         actual -> actual.equalsDeep(expected),
         expected.toString());
   }
+  public static Matcher<Resource> isFhir(Resource expected) {
+    return new FunctionMatcher<>(
+        actual -> actual.equalsDeep(expected),
+        expected.toString());
+  }
```

Comment:
Since resource extends base i suggest removing this and refactoring the matchers above to look like:
```suggestion
  public static <T extends Base> Matcher<T> isFhir(T expected) {
    return new FunctionMatcher<>(
        actual -> actual.equalsDeep(expected),
        expected.toString());
  }
```

Reply:
That's a good point, I didn't even notice - thank you, will update!